contrastive_vi/model/total_contrastive_vi.py

```diff
@@ -750,3 +745,55 @@ def differential_expression(
         )
 
         return result
+
+    @torch.no_grad()
+    def get_latent_library_size(
+        self,
+        adata: Optional[AnnData] = None,
+        indices: Optional[Sequence[int]] = None,
+        give_mean: bool = True,
+        batch_size: Optional[int] = None,
+    ) -> np.ndarray:
+        r"""
+        Returns the latent library size for each cell.
```

Comment:
I think the documentation would be clearer if we say "latent **RNA** library size," as some users may mistake this method as something related to protein expression.

Reply:
Good catch. Added the word "RNA" for clarity